libs/gltfio/src/AssetLoader.cpp

```diff
@@ -525,12 +537,12 @@ void FAssetLoader::createRenderable(const cgltf_data* srcAsset, const cgltf_node
         }
     }
 
-    auto& morphTargetNames = mResult->mMorphTargetNames[entity];
-    assert_invariant(morphTargetNames.empty());
-    morphTargetNames = FixedCapacityVector<CString>(numMorphTargets);
+    FixedCapacityVector<CString> morphTargetNames(numMorphTargets);
     for (cgltf_size i = 0, c = mesh->target_names_count; i < c; ++i) {
-        morphTargetNames[i] = StaticString::make(mesh->target_names[i]);
+        morphTargetNames[i] = CString(mesh->target_names[i]);
```

Comment:
we need to use a `CString` here because the `target_names[i]` is not persistant?

Reply:
Correct. This codeline was an error introduced by a 3P contribution.